xyz

```diff
@@ -157,17 +158,12 @@ for script in "${scripts[@]}" ; do
   run "VERSION=$next_version PREVIOUS_VERSION=$version '$script'"
 done
 
-inc() {
-  if [[ -f "$1" ]] ; then
```

Comment:
This guard is vestigial: it was useful back when we would update the `version` field of __bower.json__ or __component.json__ if one of these files existed. We now only update the `version` field of __package.json__.

Reply:
:mortar_board: